yamls/kube-ovn.yaml

```diff
@@ -196,28 +196,22 @@ spec:
               mountPropagation: HostToContainer
             - mountPath: /etc/localtime
               name: localtime
-          readinessProbe:
-            exec:
-              command:
-                - nc
-                - -z
-                - -w3
-                - 127.0.0.1
-                - "10665"
-            periodSeconds: 3
-            timeoutSeconds: 5
           livenessProbe:
-            exec:
-              command:
-                - nc
-                - -z
-                - -w3
-                - 127.0.0.1
-                - "10665"
+            failureThreshold: 3
             initialDelaySeconds: 30
             periodSeconds: 7
-            timeoutSeconds: 5
-            failureThreshold: 5
+            successThreshold: 1
+            tcpSocket:
+              port: 10665
+            timeoutSeconds: 1
```

Comment:
`timeoutSeconds` should be long enough, when the system load is high it may not response in 1 second